dashboard/README.md

```diff
@@ -1,198 +1,161 @@
-    │   │   ├── tableOfContentReducer.js
-    │   │   ├── toastReducer.js
-    │   │   └── userProfileReducer.js
-    │   ├── store
-    │   │   └── store.js
-    │   ├── utils
-    │   │   ├── axiosInstance.js
-    │   │   ├── dateFunctions.js
-    │   │   ├── filterDataset.js
-    │   │   ├── helper.js
-    │   │   └── routeConstants.js
-    │   ├── App.css
-    │   ├── App.js
-    │   ├── App.test.js
-    │   ├── index.css
-    │   ├── index.js
-    │   ├── logo.svg
-    │   ├── reportWebVitals.js
-    │   └── setupTests.js
-    ├── config-overrides.js
-    ├── jsconfig.json
-    ├── jsdoc.json
-    ├── package.json
-    ├── package-lock.json
-    └── README.md
+ .
+ ├── public									# HTML entry
+ ├── server 								# endpoint configuration
+ └── src
+     ├── actions							# redux actions
+     ├── assets
+     │   ├── constants						# constants used in code
+     │   ├── images							# images
+     │   └── logo							# Pbench logos
+     ├── modules
+     │   ├── components						# functional components
+     │   └── containers						# parent components
+     ├── reducers							# redux reducer
+     ├── store								# redux store
+     └── utils								# utility functions
 ```
 
+  
+
 ## Assets
 
+  
+
 Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
 
+  
+
 ## Prerequisites
 
+  
+
 ## Install Node JS
-Refer to https://nodejs.org/en/ to install nodejs
+
+Refer [https://nodejs.org/en/] (Node.js) official website
+
+  
 
 ## Install create-react-app
+
 Install create-react-app npm package globally. This will help to easily run the project and also build the source files easily. Use the following command to install create-react-app
 
+  
+
 ```bash
+
 $ npm install -g create-react-app
+
 ```
 
-## Cloning and Running the Application in local
+  
+
+## Cloning and Running the Application in local  
+
+- Clone the [Pbench code](https://github.com/distributed-system-analysis/pbench) into local
+- Install all the npm packages
 
-Clone the project into local
-Install all the npm packages. Go into the project folder and type the following command to install all npm packages
+Go into the project folder and type the following command to install all npm packages 
 
 ```bash
+
 $ npm install
+
 ```
-In order to run the application Type the following command
+  
+
+`server/index.js` contains references to datastores and metadata required to visualize data in the dashboard.
+
+  
+
+Create `.env` file in the root directory and declare the environment variable
+
+  
 
 ```bash
+
+$ PBENCH_SERVER=http://test_domain.com
+
+```
+
+where `PBENCH_SERVER` should point to a real server.
+
+In order to start the express server and run the application use the following command 
+
+```bash
+
 $ npm run dev
+
 ```
-The Application Runs on localhost:3000
+
+
+
+The application runs on http://localhost:3000 in the default browser.
```

Comment:
[You said](https://github.com/distributed-system-analysis/pbench/pull/2963#discussion_r953677539):
> It automatically loads on the default browser.

We should say something more like that, here.  (Being a little more explicit will make the meaning much clearer.)

Reply:
No, it basically serves the Javascript code from localhost:3000 so that you can connect to it with a browser. You can then connect to it with any browser. The real point is that npm can fire up background express/db services to emulate a normal UI application environment.

But, yeah, the "in the default browser" is really incorrect and should be dropped.

Reply:
I'm guessing that, if one were to run this on a platform which had an installed browser configured with an accessible graphics display, then, in fact, it would do the local magic to cause the browser to run and load the Dashboard script...I've just never run it like that.  😆

So, I think this text needs to be expanded along the lines that you suggest, Dave, saying that npm brings up a server which can be hit at `localhost:3000` (and, I presume at 192.168.1.200:3000 or whatever the appropriate IP address is) which emulates the "real" server.

Reply:
Dropped "in the default browser"